cmd/kube-proxy/app/server_others_test.go

```diff
@@ -67,7 +67,7 @@ func Test_getProxyMode(t *testing.T) {
 		},
 		{ // detect, error
 			flag:          "",
-			iptablesError: fmt.Errorf("oops!"),
+			iptablesError: fmt.Errorf("oops"),
```

Comment:
I like to use errors.New if there's no actual arguments to the error; but it doesn't really matter much and this is an error just for the test anyway.

Reply:
I wasn't sure enough about best-practices there to suggest a change, but I usually do that too.